lms/djangoapps/oauth_dispatch/views.py

```diff
@@ -0,0 +1,87 @@
+"""
+Views that dispatch processing of OAuth requests to django-oauth2-provider or
```

Comment:
Can we expand this docstring so it's clear that the dispatching is only for Application-based dispatching, and not Access Token based dispatching?